README.md

```diff
@@ -197,6 +197,24 @@ Key Storage | 60405 | Exception while storing/loading key pairs
 Initialization | 60406 | Exception while initializing an object
 Authentication Token | 60407 | Exception while generating token
 
+### Getting server error
+You can control custom server errors listed here (https://www.twilio.com/docs/api/errors) by following the next example:
+
+```swift
+/// example error retrieved when creating a factor.
+let error = TwilioVerifyError()
```

Comment:
Can you use the code snippet to create a factor instead?
```
twilioVerify.createFactor(withPayload: payload, success: { factor in
  // Success
}) { error in
  // Error
}
```
Replacing // Error with the next lines

Reply:
Also, we can add a link to the example in the sample app: https://github.com/twilio/twilio-verify-ios/blob/main/TwilioVerifyDemo/TwilioVerifyDemo/CreateFactor/Presenter/CreateFactorPresenter.swift#L50

It will work when it is merged

Reply:
Done!

https://github.com/twilio/twilio-verify-ios/blob/cmejia/22264/error/README.md